packages/esm-ohri-pmtct-app/src/utils/pmtct-helpers.ts

```diff
@@ -0,0 +1,26 @@
+import { getIdentifierInfo } from '../api/api';
+
+export const generateInfantPTrackerId = (fieldId: string, motherPtrackerId: string): string | undefined => {
+  if (!fieldId || !motherPtrackerId) return;
+  return fieldId === 'infantPtrackerid'
+    ? motherPtrackerId + '1'
+    : fieldId.includes('_')
+    ? motherPtrackerId.concat((Number(fieldId.split('_')[1]) + 1).toString())
+    : undefined;
+};
+
+export const getIdentifierAssignee = async (identifier: string, identifierType: string) => {
+  return getIdentifierInfo(identifier).then((data) => {
+    if (data) {
+      for (const result of data.results) {
```

Comment:
What happens if `data` has no `results` attribute?

Reply:
This comment also applies to the lines below, you better explicitly check or add the question marks

Reply:
In this case, by the time we get here, all the other possible errors were checked and the call is fine. If there's no data, it only means that there's no patient with that identifier meaning it's ok to go ahead and assign it to the current patient.